File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/workbook/CreateEmptyWorkbook.java
package com.aspose.cells.cloud.examples.workbook;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;
import java.util.UUID;

public class CreateEmptyWorkbook {

	public static void main(String... args) throws IOException {
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);
			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);

			String output = "Sample-" + UUID.randomUUID().toString() + ".xlsx";
			Path outputFile = Utils.getPath(CreateEmptyWorkbook.class, output);

			cellsApi.PutWorkbookCreate(output, null, null, Utils.STORAGE, null, null);

			com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload(output, null, Utils.STORAGE);

			Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);
		} catch (Exception e) {
			e.printStackTrace();
		}

	}
}

File: Examples/Node.js/SDK/workbook/CreateEmptyWorkbook.js
var fs = require('fs'),
    assert = require('assert'),
    utils = require('../utils.js'),
    cells = utils.cellsApi,
    storage = utils.storageApi;



var outputName = "Output-" + Date.now() + ".xlsx";
var outputPath = utils.getPath(__filename, outputName);

cells.PutWorkbookCreate(outputName, null, null, null, null, null, function(responseMessage) {
    assert.equal(responseMessage.status, 'OK');
    //download updated Workbook from storage server
    storage.GetDownload(outputName, null, null, function(responseMessage) {
        assert.equal(responseMessage.status, 'OK');
        fs.writeFileSync(outputPath, responseMessage.body);
        console.log(outputPath, "workbook has been created successfully");
    });
});
